### PR TITLE
add finance instructions to wato wiki

### PR DESCRIPTION
**Pull request by victorzheng02**
Adds instructions on how to use the finance instructions to the wato wiki.

### CODE REVIEW

File: pages/finance/creating_personal_purchases.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Creating Personal Purchases
+
+1. Go to https://finance-frontend.watonomous.ca/
+2. Create New Ticket > Ticket Type: Personal Purchase
+3. Enter all purchase details. For funding item link, you would either be using FI-1 (WATO Cash) or one of the Funding Items that match the category. For example, if you're purchasing an RTX 4090 it would probably fit under GPU funding.
+4. Await team approval. Please reach out to your team director, team captain, and the faculty advisor.
```
Comment (ansonjwhe) [inline pages/finance/creating_personal_purchases.mdx:6]: does "team director, team captain, and faculty advisor" still make sense? would "finance team and faculty advisor" make more sense?

File: pages/finance/creating_personal_purchases.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+5. Once this has been completed, the item will be sent to the finance coordinator. They will purhcase the item, in which case the status will be `ORDERED`
```
Comment (ansonjwhe) [inline pages/finance/creating_personal_purchases.mdx:7]: typo at "purhcase"

File: pages/finance/creating_purchase_requests.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Creating a University of Waterloo Finance reimbursement form.
+
+1. Navigate [here](https://uwaterloo.ca/engineering-endowment-foundation/getting-funding/spending-funding/submitting-reimbursement-request)
+2. Click the link in step 1 titled "reimbursement form (excel)"
+3. Enter the following fields
+4. Payee is Student
+5. Claimant Name, Student Number, Department, Phone Number, E-mail Address
+6. Mailing Address, City, Province/State, Postal Code. For these, use the address your items were shipped to, or your personal address.
+7. Destination/Reason for Request: Please put the corresponding reference number. Reach out to @smileycow on Discord if you do not know what this is
```
Comment (ansonjwhe) [inline pages/finance/creating_purchase_requests.mdx:19]: for my knowledge, is the reference number the PPR code or is it something else?

**Review by ansonjwhe — APPROVED** [review]
lgtm! few small comments

File: pages/finance/creating_purchase_requests.mdx (hunk 3 above)
Comment (victorzheng02) [reply pages/finance/creating_purchase_requests.mdx:19]: this is the reference code from weef

**Pull request merged**